M2/Macaulay2/packages/Macaulay2Doc/ov_top.m2

```diff
@@ -10,57 +10,129 @@ Node
     Code
       IMG ("src" => replace("PKG", "Style", currentLayout#"package") | "9planets.gif", "alt" => "image of nine planets")
     Text
-      Macaulay2 is a software system devoted to supporting research in algebraic geometry and commutative algebra,
-      developed with funding from the National Science Foundation. We are eager to help new users get started with it.
-  Subnodes
-    :Preface
-      "Copyright and license"
-      "Acknowledgements"
-      "The authors"
-      -- TODO: node to be written, will point to our book, Hal's book, Mike's Arizona notes, Sage, etc:
-      "Other sources of information about Macaulay2"
-    :For the New User
-      "getting started"
-    :Mathematical Objects
-      "rings"
-      "ideals"
-      "matrices"
-      "modules"
-      "morphisms"
-      "chain complexes"
-      "Varieties :: Varieties"
-    :Some Mathematical Computations
-      "mathematical tutorials"
-      "Gröbner bases"
-      "normal forms"
-      -- Mike wanted this: TO "Hilbert functions"
-      "elimination of variables"
-      -- Mike wanted this: TO "syzygies"
-      -- Mike wanted this: TO "saturation"
-      -- Mike wanted this: TO "fibers of a map between varieties"
-      -- Mike wanted this: TO "solving systems of polynomial equations"
-      "IntegralClosure :: IntegralClosure"
-      "PrimaryDecomposition :: primaryDecomposition"
-      --
-      "arithmetic functions"
-      "commutative algebra functions"
-      "homological algebra functions"
-      "algebraic geometry functions"
-      "combinatorial functions"
-      "analytical functions"
-    :The Macaulay2 Language
-      "The Macaulay2 language"
-    :Packages
-      "packages provided with Macaulay2"
-      "authors of Macaulay2 packages"
-    :Appendix
-      "changes to Macaulay2, by version"
-      "basic commutative algebra"
-      -- Mike wanted this: TO "frequently encountered problems"
-      "replacements for commands and scripts from Macaulay"
-    :Reference material
-      "Thing"
-      "Type"
-      "ImmutableType"
-      "Function"
+      Macaulay2 is an interpreted, dynamically typed programming language designed to
+      support research in commutative algebra, algebraic geometry and related fields.
+      All components of the language are open sourced, including over two hundred
+      contributed packages, and generously funded by the National Science Foundation.
+    Tree
+      :Getting Started
```

Comment:
Can you explain the (for me) new syntax here?  (:, > ). Where is the doc for it?